03LessonThree/homework/git-scm.html

```diff
@@ -30,15 +30,15 @@
                     <li>Feature Based Workflow. Create new branches for each new feature you're working on so you can seamlessly switch back and forth between them, then delete each branch when that feature gets merged into your main line.</li>
                     <li>Disposable Experimentation. Create a branch to experiment in, realize it's not going to work, and just delete it - abandoning the work—with nobody else ever seeing it (even if you've pushed other branches in the meantime).</li>
                 </ul>
-            </div>
-            <div id="article">
+            </article>
+            <article>
                 <h3>Community</h3>
                 <p>Questions or comments for the Git community can be sent to the mailing list by using the email address git@vger.kernel.org. Bug reports should be sent to this mailing list.</p>
                 <p>You do not need to subscribe: you will be Cc'd in replies. Please keep the Cc list intact when replying (use "Reply to all"). Greylisting may delay your first post for a few hours. Note also that the list only accepts plain-text email; please disable HTML in your outgoing messages.</p>
                 <p>The archive can be found on Gmane. Click here to subscribe. If you have trouble sending to the list with your regular mail reader, you may also post via gmane. There is also Git user mailing list on Google Groups which is a nice place for beginners to ask about anything.</p>
-            </div>
-        </div>
-        <div id="aside">
+            </article>
+        </section>
+        <aside>
             <div>
             <h4>Rapid development with FTS</h4>

```

Comment:
This could have been indented as it is inside the `<div>` container.
Otherwise, you have done a fantastic job! Thank you.